pkg/ansible/watches/watches.go

```diff
@@ -159,36 +159,50 @@ func (w *Watch) addRolePlaybookPaths() {
 	}
 }
 
-// getFullPath will return a valid full path for the playbook
+// getFullPath will return a valid full role for the playbook
 func getFullPath(path string) string {
 	if len(path) > 0 && !filepath.IsAbs(path) {
 		return filepath.Join(projutil.MustGetwd(), path)
 	}
 	return path
 }
 
-// getFullRolePath will return a valid full path for the role
+// getFullRolePath will return a valid full role for the role
 func getFullRolePath(path string) string {
 	if len(path) > 0 && !filepath.IsAbs(path) {
 		envVar, ok := os.LookupEnv("ANSIBLE_ROLES_PATH")
 		if ok && len(envVar) > 0 {
-			return filepath.Join(envVar, path)
+			// Check all values informed (e.g. path1/roles:path2/roles:/absolute/path3/roles)
+			result := strings.Split(envVar, ":")
+			for i := range result {
+				// Check if the role can be found in the envVar + role role
+				infoPath := filepath.Join(result[i], path)
+				if _, err := os.Stat(infoPath); err == nil {
+					return infoPath
+				}
+
+				// Check if the role can be found in the envVar + roles + role role
+				infoPathWithRolesDir := filepath.Join(result[i], "roles", path)
+				if _, err := os.Stat(infoPathWithRolesDir); err == nil {
+					return infoPathWithRolesDir
+				}
```

Comment:
Does ANSIBLE_ROLES_PATH work this way (where it check is in `<path>/<role>` and `<path>/roles/<role>`)?

Reply:
The Ansible documentation do not explain how it should work. 
However, according to it maintainer

> A role should be found in the defined "roles path" (via config) or in a roles/ directory adjacent to the play
From: https://github.com/ansible/ansible/issues/16911#issuecomment-236765123